doc/source/User_guide/pyaedt_file_data/cable.rst

```diff
@@ -0,0 +1,23 @@
+Cable file
+==========
+
+The cable configuration file allows to create a cable geometry with shielding from a JSON file.
```

Comment:
```suggestion
The cable configuration file allows you to create a cable geometry with shielding from a JSON file.
```
I think the "from a JSON file" might be misplaced. Are you using the JSON file to create a cable geometry with shielding? Are we talking two files here--a configuration file and JSON file. I think this sentence should be rewritten.